engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala

```diff
@@ -324,7 +324,7 @@ class MainModule(serverConfig: LanguageServerConfig, logLevel: Level) {
     .logLevel(logLevel)
     .strictErrors(false)
     .disableLinting(false)
-    .enableIrCaches(true)
+    .enableIrCaches(false) // Enable once #11022 is addressed
```

Comment:
Is the sole reason for this change the #11022 problem? Then don't disable the caching!

Reply:
No, the motivation is #11020. In general I prefer to revert/disable problematic changes if a) I don't know how long it will take to fix a bug b) if other devs are regularly encountering them in day-to-day activities, thus obscuring the state of the product.